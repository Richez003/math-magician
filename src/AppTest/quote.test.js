import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../page/quote';

it('renders correctly', () => {
  const tree = renderer.create(<Quotes />).toJSON();
  expect(tree).toMatchSnapshot();
});
