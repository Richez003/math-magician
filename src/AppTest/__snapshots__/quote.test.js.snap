// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main>
  <div>
    <p>
      Mathematics is not about numbers, equations, computations or algorithms: It is about understanding.
    </p>
    <br />
    <p>
      - William Paul, Thurston
    </p>
  </div>
</main>
`;
